web/themes/joinup/templates/search_api/facets-type-wrapper--more.html.twig

```diff
@@ -12,8 +12,13 @@
  * @ingroup themeable
  */
 #}
+{% if items|length > 2 %}
+  {% set class='tab__container tab__container--wide' %}
```

Comment:
No problem. Do we know how items should behave in specific number and in specific breakpoint? For example on desktop right now 3 items are aligned to the left with small spacing, but 4 are evenly spread over the entire length of the container.